javascript-demo/index.html

```diff
@@ -13,6 +13,10 @@
   <script type="module" src="comparisons.js"></script>
   <script type="module" src="strings.js"></script>
   <script type="module" src="arrays.js"></script>
+  <script type="module" src="loops.js"></script>
+  <script type="module" src="functions.js"></script>
+  <script type="module" src="arrowFunctions.js"></script>
+  <script type="module" src="objects.js"></script>
```

Comment:
Only import function.js file in this MR. Please remove loops.js, arrowFunctions.js, and objects.js